### PR TITLE
Fixed menu refreshing

### PR DESCRIPTION
**Pull request by amit-sides**
I fixed the welcome screen and menu so that they refresh the screen only when it needs to update the display.
Tested on my badge and seems to work fine.

### CODE REVIEW

File: code.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def main_screen():
 last_addon = None
 while True:
     if refresh_counter and not badge.display.time_to_refresh:
-        main_screen()
+        # main_screen()
```
Comment (urish) [inline code.py]: Is there a reason for keeping this commented out and not removing altogether?

Comment (amit-sides) [reply code.py]: I didn't check if refresh_counter is used anywhere else actually, just wanted to check if commenting out the function that calls `badge.display.refresh()` will fix the unnecessary refreshes at the start. Probably can be removed...

Comment (urish) [reply code.py]: so let's remove it... less code = less bug :)

Comment (amit-sides) [reply code.py]: I removed it and tested again, it seems to work fine as well!

Comment (urish) [reply code.py]: Cool! So are we good to merge?

Comment (amit-sides) [reply code.py]: I believe so!

Comment (urish) [reply code.py]: Let's!

**Review by urish — APPROVED** [review]

**Pull request merged**